Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'feedback'
+DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
+
+node('mongodb-2.4') {
```

Comment:
https://ci.dev.publishing.service.gov.uk/job/govuk_feedback/configure doesn't restrict where this build should be run so I think we can omit the mongodb-2.4 tag here